test/index-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chai = require('chai');
+const main = require('../index.js');
+const assert = chai.assert;
```

Comment:
Good job using assert. FYI that most of the time, Duncan is likely to use the expect method for our future testing as it's considered more human readable. It's good to play around with different testing methods to become familiar with it though.